test/e2e/lib/mocha-hooks.js

```diff
@@ -51,8 +52,51 @@ afterEach( function () {
 	}
 } );
 
-// Take Screenshot
+// Take Screenshot for Playwright.
 afterEach( async function () {
+	if ( ! global.isPlaywright ) {
```

Comment:
(For a future PR, noting it here so we don't forget)

This can use some refactoring. I think we can have a `mocha/hooks/` with many individual hooks (eg: `mocha/hook/playwright-screenshot.js`), an index that imports some hooks (eg: `mocha/playwright-hooks.js`), and specify it inside the `magellan-*.json` config file with `mocha_args: '--file mocha/playwright-hooks.js'`

Reply:
Interesting, I had not known about this ability and quick search with relevant keywords don't turn up any results. It might entrench our reliance on Magellan for a bit longer, but that would help me remove a bunch of unclean if/else statements from the hook code.